index.js

```diff
@@ -30,6 +30,56 @@ const getHeadSha = async (repo) => {
   return data.commit.sha;
 }
 
+async function closePRs() {
+   // Close old auto PRs
+   const {data: prs} = await octokit.pulls.list({
+     owner: 'cds-snc',
+     repo: 'notification-tf',
+     state: 'open'
+   });
+
+   prs.forEach( async pr => {
+     if(pr.title.startsWith("[AUTO-PR]")) {
+       await octokit.pulls.update({
+         owner: 'cds-snc',
+         repo: 'notification-tf',
+         pull_number: pr.number,
+         state: "closed"
+       });
+       await octokit.git.deleteRef({
+         owner: 'cds-snc',
+         repo: 'notification-tf',
+         ref: `heads/${pr.head.ref}`
+       });
+     }
+   })
+}
+
+async function hasChangesInStaging() {
```

Comment:
What do you think of naming this differently? It's only checking for staging TF pull requests and not checking the staging environment